sdk/ai/Azure.AI.Projects/CHANGELOG.md

```diff
@@ -4,10 +4,16 @@
 
 ### Features Added
 
+* Added `AzureFunctionToolDefinition` support to inform Agents about Azure Functions.
+* Added `OpenApiTool` for Agents, which creates and executes a REST function defined by an OpenAPI spec.
+* Add `parallel_tool_calls` parameter to allow parallel tool execution for Agents.
```

Comment:
Could you please mention the method names where you added the `parallelToolCalls` parameter?  

Also, a small nit: please update the parameter name from `parallel_tool_calls` to `parallelToolCalls`.

Reply:
Fixed.